### PR TITLE
Add sinatra-sequel-puma-rbx benchmark

### PR DESCRIPTION
**Pull request by mwpastore**
Add a new version of the sinatra-sequel framework that runs on Puma and Rubinius.

### CODE REVIEW

**Comment by msmith-techempower** [comment]
Relies on #1794 being resolved, then needs to be rebased and retested.

**Comment by msmith-techempower** [comment]
@mwpastore We have updated our toolset to fix the `rvm` issues on Travis-CI, but we need you to rebase and push for a test to kick off.

**Comment by mwpastore** [comment]
Excellent, will do.

**Comment by msmith-techempower** [comment]
Additionally, [Travis-CI](https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/97061768) is reporting errors.

JRuby is not working correctly, but it is hard to tell if that's endemic of the Travis-CI environment or an actual problem with implementations since #1789 is having similar issues with JRuby.

RBX seems to completely fail at compilation.

**Comment by msmith-techempower** [comment]
This needs to be rebased and retested since I have merged in #1789

File: toolset/setup/linux/client.sh [hunk 1]
```diff
@@ -21,9 +21,9 @@ sudo sh -c "echo '*               -    nofile          65535' >> /etc/security/l
 # wrk
 ##############################
 
-git clone https://github.com/wg/wrk.git
-cd wrk
-git checkout 205a1960c8b8de5f500bb143863ae293456b7add
+curl -sL -o wrk-4.0.1.tar.gz https://github.com/wg/wrk/archive/4.0.1.tar.gz
```
Comment (msmith-techempower) [inline toolset/setup/linux/client.sh]: Please undo the changes to this file so that it is not included in your pull request.

**Comment by msmith-techempower** [comment]
Travis couldn't detect a change in the rebase - can you add/remove a space to `frameworks/Ruby/sinatra-sequel/hello_world.rb` and commit it so that Travis-CI kicks off another run and hopefully detects `sinatra-sequel` needs to be tested?

File: toolset/setup/linux/client.sh (hunk 1 above)
Comment (mwpastore) [inline toolset/setup/linux/client.sh]: Hah, thanks, I was quite confused a few seconds there.

**Comment by msmith-techempower** [comment]
Well, jruby is still broken without much information as to why, but I still think that that is unrelated, and since the one test passed, I'm inclined to merge this in.

**Comment by mwpastore** [comment]
Thanks Mike. I have Travis set up on my fork so I'll see if I can figure out a solution for JRuby. Cheers!
